hubble/Makefile

```
@@ -62,5 +61,20 @@ local-release: clean
 		rm -r release/$$OS; \
 	done;
 
+install: install-binary install-bash-completion-only
+
+install-binary:
```

this depends on the hubble cli to be built right?

```suggestion
install-binary: $(TARGET)
```